src/main/webapp/app/exam/monitor/exam-monitoring.service.ts

```diff
@@ -20,51 +19,63 @@ export class ExamMonitoringService {
         if (!monitoring) {
             return;
         }
-        const examActivity = studentExam.examActivity || new ExamActivity();
-        examAction.timestamp = this.serverDateService.now();
-        examActivity.addAction(examAction);
-        studentExam.examActivity = examActivity;
+        try {
+            const examActivity = studentExam.examActivity || new ExamActivity();
+            examAction.timestamp = this.serverDateService.now();
+            examAction.studentExamId = studentExam.id;
+            examActivity.addAction(examAction);
+            studentExam.examActivity = examActivity;
+        } catch (e) {}
```

Comment:
Should we send the error to sentry?

Reply:
I guess no - since we dont want any influence on the exam, we can just ignore those errors :+1: